modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -1,77 +1,360 @@
+This template is a simplified version of the configuration. The final configuration of the template can vary depending on the controller and ingress routing rules you use. Review the documentation for your ingress controller carefully.
 ====
 
-== Procedure
+[source,yaml]
+--
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: example-rtf-ingress
+  namespace: rtf #<1>                           
+  labels:
+    business-group: business-group-id
+    environment: environment-id
+spec:
+  ingressClassName: rtf-nginx #<2> 
+  tls: #<4>
+  - hosts:
+      - app-name.example.com
+    secretName: example-tls
+  rules:
+  - host: app-name.example.com #<3>
+    http:
+      paths: #<5>
+      - pathType: Prefix
+        path: /
+        backend: #<6>
+          service:
+            name: service-name
+            port:
+              name: service-port
+--
 
-. Copy the following into a text file:
+Note the following about this example:
+  
+<1>  The template must be placed in the `rtf` namespace.
+
+<2>  ingressClassName must be prefixed with `rtf-`, for example, `rtf-nginx`.
 +
+This is how Runtime Fabric recognizes the object as a template. A template that uses the `rtf-` prefix in the `ingressClassName`, for example, `rtf-nginx`, is consumed by the Runtime Fabric agent only and not by the actual ingress controller. Ingress controllers discover only resources with an `ingressClassName` value that uses the vendor-specific name, for example, `nginx` or `haproxy`.
+
+<3> TLS is optional.
+
+<4> `app-name` is a placeholder parameter that will be replaced with actual app name when you deploy the application. It helps to ensure that each endpoint name is unique if there is not a wildcard in the subdomain.
+
+
+<5> A template can include multiple hosts, but Runtime Manager displays the first path rule only.
+
+<6>  These are placeholder values required for Kubernetes validation, but the actual values are not used by Runtime Fabric. 
+
+For a path-based routing ingress template, consult your ingress controller documentation and configure it accordingly.
+For an example using Nginx, see https://help.mulesoft.com/s/article/Path-based-Nginx-Ingress-Routing-Example-for-Runtime-Fabric-on-Self-Managed-Kubernetes[Path-based Nginx Ingress Routing Example for Runtime Fabric on Self-Managed Kubernetes].
+
+=== networking.k8s.io/v1beta1
+
+[source,yaml]
+--
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+  - host: app-name.example.com
+    http:
+      paths: #<1>
+      - path: /
+        pathType: 
+        backend: #<2>
+          serviceName: service
+          servicePort: 80
```

Comment:
This example is copy-paste accurate, i.e. if a customer copy pastes and attempts to kubectl apply this, it will fail.